kedro/contrib/io/azure/csv_blob.py

```diff
@@ -35,10 +35,11 @@
 import pandas as pd
 from azure.storage.blob import BlockBlobService
 
+from kedro.contrib.io import DefaultArgumentsMixIn
 from kedro.io import AbstractDataSet
 
 
-class CSVBlobDataSet(AbstractDataSet):
+class CSVBlobDataSet(AbstractDataSet, DefaultArgumentsMixIn):
```

Comment:
Base class method resolution proceeds from left to right, so Mixins should always be to the left and base/abstract classes to the right.

Reply:
@tolomea Thanks for pointing this out! Resolved, although I assume we should also do the same for `FilepathVersionMixIn`/`S3PathVersionMixIn` in another PR.